files/jquery.dirtyforms/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "jquery.dirtyforms",
+  "repo": "NightOwl888/jquery.dirtyforms.dist",
+  "files": {
+    "include": ["jquery.dirtyforms.min.*", "plugins/*.min.*"]
```

Comment:
Hey, do you not want to include the `.map` files? I would be fine with you just including all the `js` and `map` files in your repo